nextflow.config

```diff
@@ -48,326 +55,584 @@ if ( params.medaka || params.nanopolish || params.viridian ){
-      }
-          withName:makeQCCSV {
-              time = '1h'
-              singularity.enabled = true
-              container = "/data/images/artic-ncov2019-illumina_v1.10.0.sif"
-      }
-          withName:writeQCSummaryCSV {
-              time = '1h'
-              singularity.enabled = true
-              container = "/data/images/artic-ncov2019-illumina_v1.10.0.sif"
-      }
-          withName:collateSamples {
-              time = '1h'
-              singularity.enabled = true
-              container = "/data/images/artic-ncov2019-illumina_v1.10.0.sif"
-      }
-          withName:getObjFiles {
-              time = '30m'
-              singularity.enabled = true
-              container = '/data/images/oci_pipeline_v0.1.14.sif'
-      }
-          withName:pango {
-              time = '5m'
-              singularity.enabled = true
-              container = '/data/images/pango_v0.1.14.sif'
-      }
-          withName:nextclade {
-              time = '5m'
-              singularity.enabled = true
-              container = '/data/images/nextclade_v0.1.14.sif'
-      }
-          withName:download_nextclade_files {
-              time = '1h'
-              singularity.enabled = true
-              container = '/data/images/nextclade_v0.1.14.sif'
-      }
-          withName:getVariantDefinitions {
-              time = '1h'
-              singularity.enabled = true
-              container = '/data/images/aln2type_v0.1.14.sif'
-      }
-          withName:aln2type {
-              time = '5m'
-              singularity.enabled = true
-              container = '/data/images/aln2type_v0.1.14.sif'
-      }
-          withName:viridian {
-              time = '45m'
-              singularity.enabled = true
-              container = '/data/images/viridian_workflow_v0.3.0.img'
-      }
-          withName:viridianPrimers {
-              time = '45m'
-              singularity.enabled = true
-              container = '/data/images/viridian_workflow_v0.3.0.img'
-      }
-          withName:viridianAuto {
-              time = '45m'
-              singularity.enabled = true
-              container = '/data/images/viridian_workflow_v0.3.0.img'
-      }
-          withName:FN4_upload {
-              time = '1h'
-              singularity.enabled = true
-              container = '/data/images/fn4_upload_v0.1.14.sif'
-      }
-          withName:uploadToBucket {
-              time = '1h'
-              singularity.enabled = true
-              container = '/data/images/fn4_upload_v0.1.14.sif'
-      }
-          withName:makeReport {
-              time = '1h'
-              singularity.enabled = true
-              container = "/data/images/artic-ncov2019-illumina_v1.10.0.sif"
-    }
-          withName:getGFF3 {
-              time = '1h'
-              singularity.enabled = true
-              container = "/data/images/bcftools_v0.1.14.sif"
+        docker.registry = 'lhr.ocir.io/lrbvkel2wjot'
```

Comment:
Todo: Try to use default docker registry instead of OCIR. If we do so, we will need to update the path of the default container image in line 416 and line 517, plus we need to make sure all images we pull from docker has a unique tag for every single release.